ynr/apps/elections/uk/templatetags/home_page_tags.py

```diff
@@ -141,3 +141,27 @@ def results_progress(context):
 
 
     return context
+
+
+@register.inclusion_tag('includes/by-election-ctas.html',
+                        takes_context=True)
+def by_election_ctas(context):
+    context['SHOW_BY_ELECTION_CTA'] = getattr(
+        settings, 'SHOW_BY_ELECTION_CTA', False)
```

Comment:
Do you think we should set a default value for this in `local.py.example` in this PR?

Also, in the case where this is `False`, can we hide the `<div>` completely so that we don't show an empty box:

![screenshot at 2018-05-30 20 45 25](https://user-images.githubusercontent.com/6025893/40744514-a9b9ffb6-644c-11e8-9f42-5d9e80c875ce.png)

Reply:
This isn't a new problem in this PR – there are a few things that could be in that box, and basically one of them always is. Currently there is `SHOW_ELECTION_STATS`,  `SHOW_SOPN_TRACKER`,  `SHOW_RESULTS_PROGRESS` and  `SHOW_BY_ELECTION_CTA`. Any any point, one of these will be `True`, but there isn't a process for ensuring that exactly one is True at the moment.

What do you think a good way of doing this would be?

Reply:
Maybe another way to look at it would be that if we always want exactly one thing to be true, then we don't want several boolean settings, we want one setting (e.g: `FRONTPAGE_CTA`) which is a string like `sopn_import_progress` or `current_election_stats` that tells us what template to include? If that setting is missing/invalid, we don't show anything.

Given this is unrelated to the changes in this PR, happy to address as a different issue to avoid scope creep in this PR.

Reply:
Right, that's nice. Happy to make that change in this PR as it's not especially important that this is merged quickly, so we may as well pay off some debt while we're here and talking about it.